home/private_dot_local/lib/agent-equipment/agent_equipment/updater.py

```
@@ -0,0 +1,697 @@
+        routes = _mutable_array(
+            provider_selection.get("routes"),
+            "catalog coverage routes",
+        )
+        for value in routes:
+            route = _mutable_object(value, "catalog coverage route")
+            if route.get("distribution") != distribution_identity:
+                continue
+            if route.get("restore") != old_restore:
+                raise ValueError(
+                    "update catalog route restore does not match its base manifest"
+                )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)